pyiron/lammps/base.py

```diff
@@ -212,15 +212,20 @@ def set_input_to_read_only(self):
 
     def validate_ready_to_run(self):
         """
-
-        Returns:
-
+        Validating input parameters before LAMMPS run
         """
         super(LammpsBase, self).validate_ready_to_run()
         if self.potential is None:
             raise ValueError(
                 "This job does not contain a valid potential: {}".format(self.job_name)
             )
+        scaled_positions = self.structure.get_scaled_positions()
```

Comment:
`get_scaled_positions(wrap=False)`

Reply:
I mean, it's unlikely that this is required, but I guess it's safer.

Reply:
Done!